tests/pages/checkout_page.py

```
@@ -176,50 +213,203 @@ def add_item(self, item_name, quantity=1):
+        1. open the same confirm modal as checkout, or
+        2. save directly without showing confirm.
+        """
+        try:
+            btn = self.wait_for_clickable(self.locators.CONFIRM, timeout=timeout)
+            self.driver.execute_script(
+                "arguments[0].scrollIntoView({block:'center'});", btn
+            )
+            self.driver.execute_script("arguments[0].click();", btn)
+            print("Confirm button clicked")
+            return True
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)